modules/about/ref-supported-platforms.adoc

```diff
@@ -6,10 +6,4 @@
 [id="supported-platforms_{context}"]
 = Supported platforms
 
-{product} runs on {ocp-short} {ocp-version-min}-{ocp-version} on supported architectures and providers, such as:
-
-* link:{installing-on-ocp-book-url}[{ocp-brand-name} {ocp-version-min}-{ocp-version}]
-* link:{installing-on-eks-book-url}[{eks-brand-name}]
-* link:{installing-on-aks-book-url}[{aks-brand-name}]
-* link:{installing-on-osd-on-gcp-book-url}[{gcp-brand-name}]
-* link:{installing-on-gke-book-url}[{gke-brand-name}]
+The supported features, and life cycle dates for {product}, current and past versions, are available at the link:https://access.redhat.com/support/policy/updates/developerhub[Life Cycle page].
```

Comment:
I'd suggest:

You can find the supported features and life cycle dates for both current and past versions of Red Hat Developer Hub on the Life Cycle page.